src/components/pages/Offer/OfferDetails/Stocks/StockItem/StockItem.jsx

```diff
@@ -5,10 +5,11 @@ import React, { Fragment, useCallback, useEffect, useState } from 'react'
 import Icon from 'components/layout/Icon'
 import { DateInput } from 'components/layout/inputs/DateInput/DateInput'
 import { TimeInput } from 'components/layout/inputs/TimeInput/TimeInput'
-import { DeleteStockConfirmation } from 'components/pages/Offer/Stocks/DeleteStockConfirmation/DeleteStockConfirmation'
 import * as pcapi from 'repository/pcapi/pcapi'
 
-const StockItem = ({
+import { DeleteStockConfirmation } from '../DeleteStockConfirmation/DeleteStockConfirmation'
+
+export const StockItem = ({
```

Comment:
```suggestion
const StockItem = ({
```

Un export est apparu sans suppression de l'export default à la fin